tests/foreman/performance/test_candlepin_concurrent_delete.py

```diff
@@ -0,0 +1,129 @@
+"""Test class for concurrent subscription deletion"""
+from datetime import date
+from robottelo.common import conf, ssh
+
+from robottelo.performance.perf_concurrent_test import ConcurrentTestCase
+
+
+class ConcurrentDeleteTestCase(ConcurrentTestCase):
+    """Concurrent Deleting subscriptions tests"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(ConcurrentDeleteTestCase, cls).setUpClass()
+        # parameters for concurrent activation key test
+        cls.savepoint = conf.properties[
+            'performance.test.savepoint2_enabled_repos']
+        cls.raw_file_name = conf.properties[
+            'performance.csv.raw_ak_file_name']
+        cls.stat_file_name = conf.properties[
+            'performance.csv.stat_ak_file_name']
+
+        # add date to csv files names
+        today = date.today()
+        cls.raw_file_name = '{0}-{1}'.format(
+            today.strftime('%Y%m%d'), cls.raw_file_name)
+        cls.stat_file_name = '{0}-{1}'.format(
+            today.strftime('%Y%m%d'), cls.stat_file_name)
+
+    def setUp(self):
+        super(ConcurrentTestCase, self).setUp()
+
+        # Restore database before concurrent deleting subscribed systems
+        self._restore_from_savepoint(self.savepoint)
```

Comment:
Hi guys, the point for putting resetting db here is that, since the test case would do subscription/deletion using 2,4,6,8,10 clients, but after it conducts 2 clients, it has to reset the db to clean state before 4-client test, and so on... That's why only one reset operation on base class is not enough.